type.c

```diff
@@ -491,14 +491,14 @@ int FC_FUNC( mpi_type_hvector, MPI_TYPE_HVECTOR )
   return MPI_SUCCESS;
 }
 
-int MPI_Type_hvector(int count, int blocklen, MPI_Aint stride,
+int MPI_Type_hvector(int count, int blocklength, MPI_Aint stride,
```

Comment:
Why do we need this name change (I don't have any specific preference though)?

Reply:
It is my own preference (I know it is not necessary).

Reply:
MPICH version:
```
  371 int MPI_Type_hvector(int count, int blocklength, MPI_Aint stride, MPI_Datatype oldtype,
  372                      MPI_Datatype *newtype) MPICH_API_PUBLIC;
```